kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
Why is this moved into the body of the html? This block should be in the `<head>`.